client/lib/plugins/README.md

```diff
@@ -73,59 +73,54 @@ Returns a plugin objects for a particular site.
 Returns an array of sites that have a particular plugin.
 
 
-
-
 ####Example Component Code:
 
 ```es6
 /**
  * External dependencies
  */
-var React = require( 'react' );
+import { Component } from 'react';
```

Comment:
I'm not entirely sure how much we really do need to import `React`, but there is an ESLint rule that returns an error if `React` is not in scope for a `.jsx` file. Would you mind adding it here and in subsequent occurrences?